odoo/cli/obfuscate.py

```diff
@@ -143,6 +143,9 @@ def run(self, cmdargs):
         group.add_option('--allfields', action='store_true', default=False, help="Used in unobfuscate mode, try to unobfuscate all fields. Cannot be used in obfuscate mode. Slower than specifying fields.")
         group.add_option('--vacuum', action='store_true', default=False, help="Vacuum database after unobfuscating")
         group.add_option('--pertablecommit', action='store_true', default=False, help="Commit after each table instead of a big transaction")
+        group.add_option(
+            '-y', '--yes', dest="yes", action='store_true', default=False,
```

Comment:
```suggestion
            '--perform-insecure-operation', dest="yes", action='store_true', default=False,
```

Reply:
why not `--kill-a-hundred-kitten` ?

Reply:
`--yes` is nice and just like `apt` does it (in my opinion, the short option is not needed, but can stay there).
https://linux.die.net/man/8/apt-get